ui/js/graphs.js

```diff
@@ -422,6 +425,16 @@ perf.controller('GraphsCtrl', [
         plotGraph();
       });
     }
+    
+    $scope.myMeasureChanged = function() {
+      $scope.zoom = {};
```

Comment:
if treeherder code is 4 space it should be here- but for the time being I think matching the style of the file is what it should be.  We could find a point in the near future and do a whole file 2 to 4 space PR.